alpakaConfig.h

```diff
@@ -0,0 +1,54 @@
+#ifndef AlpakaConfig_h
```

Comment:
Since we only have `using` declarations in distinct namespaces, we could avoid the `#ifndef`/`#define` protections, in case it helps simplifying the overall file structure.